.env.example

```
@@ -3,3 +3,9 @@ SOURCE_DIR="C:\Users\Username\Documents\Data_Original"
 
 # 2. まとめたファイルを保存するディレクトリ
 DEST_DIR="C:\Users\Username\Documents\Data_Flattened"
+
+# 3. 対象拡張子 (カンマ区切り、指定しない場合はデフォルト: .xlsx,.xls,.pptx,.ppt)
+TARGET_EXTENSIONS=".xlsx,.xls,.xlsm,.pptx,.ppt,.docs,.html,.pdf,.txt"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)